bin/docker/Dockerfile_ubuntu

```diff
@@ -0,0 +1,28 @@
+FROM eclipse-temurin:11-jre-jammy as runner
```

Comment:
I would just use eclipse-temurin:11-jre

Reply:
No real difference, just a bit more control over the underlayer version.